File: benchmark/results/java-invocation-inline/run-005/b-baseline/output/com/metaengine/demo/identity/services/AuthService.java
package com.metaengine.demo.identity.services;

import com.metaengine.demo.identity.aggregates.User;
import java.util.List;

/** Authentication service for the User aggregate. */
public class AuthService {

    /** Create a new User credential from a partial input. */
    public User create(User input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a User by id; returns null if not found. */
    public User findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Users, capped at the supplied limit. */
    public List<User> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a User credential by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-004/b-baseline/output/metaengine_demo/identity/services/auth_service.py
"""AuthService service module."""
from typing import Optional

from metaengine_demo.identity.aggregates.user import User


class AuthService:
    """AuthService service."""

    def create(self, input: Optional[User]) -> User:
        """Create a User."""
        raise NotImplementedError("not implemented")

    def find_by_id(self, id: str) -> Optional[User]:
        """Find a User by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: int) -> list[User]:
        """List Users."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Delete a User by id."""
        raise NotImplementedError("not implemented")
